project/Settings.scala

```diff
@@ -38,9 +39,36 @@ object Settings {
     logLevel in assembly := Level.Info,
     assemblyMergeStrategy in assembly := customMergeStrategy
   )
+  
+  lazy val dockerSettings = Seq(
+    imageNames in docker := Seq(
+        ImageName(
+          namespace = Some("broadinstitute"),
+          repository = name.value,
+          tag = Some(s"${version.value}")
+        )
+    ),
+    dockerfile in docker := {
+      // The assembly task generates a fat JAR file
+      val artifact: File = assembly.value
+      val artifactTargetPath = s"/app/${artifact.name}"
+
+      new Dockerfile {
+        from("openjdk:8")
+        expose(8000)
+        add(artifact, artifactTargetPath)
+        entryPoint("java", "-jar", artifactTargetPath)
```

Comment:
Do we need to put `-Dconfig.file = myConfig.conf` or `server` as another argument? Or can you give extra command line params at run-time?